src/components/Header/SearchBar/SearchBar.tsx

```diff
@@ -103,30 +105,39 @@ export const SearchBar = ({
           'mx-24 mt-24 md:m-0': displayLocation !== 'homepage',
         })}
       >
-        <input
-          type="text"
-          ref={textInput}
-          placeholder="Search"
-          className={cn(searchInput)}
-          value={query}
-          onChange={handleSearch}
-          style={{ ...extraInputStyle }}
-        />
-        <Icon name="icon-gui-search" size="24px" additionalCSS="absolute left-16 top-12" />
-        {!isInFocus && (
-          <div className="absolute right-16 top-12 hidden lg:flex items-center justify-end">
-            <KeyIcon className="mr-4">{keyIcon}</KeyIcon>
-            <KeyIcon className="pt-2">K</KeyIcon>
-          </div>
+        {externalScriptsData.inkeepEnabled ? (
+          <InkeepSearchBar
+            className={`${cn(searchInput)} ${cn(searchInputNoPadding)}`}
+            extraInputStyle={{ ...extraInputStyle }}
```

Comment:
Why is `extraInputStyle` being spread within an object literal here? Seems like you could just pass it in without doing that

Reply:
It was mainly from just copying the existing syntax used and it did work, so didn't question it too much. Will update :+1: